code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -784,18 +784,27 @@
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
 
 
-/datum/reagent/consumable/liquidelectricity
-	name = "Liquid Electricity"
-	description = "The blood of Ethereals, and the stuff that keeps them going. Great for them, horrid for anyone else."
+/datum/reagent/consumable/electrolyte
+	name = "Electrolytes"
+	description = "The blood of Ethereals, and the stuff necessary to carry electrical charge through their body. For everyone else, it's somewhat medicinal."
 	nutriment_factor = 5 * REAGENTS_METABOLISM
 	color = "#97ee63"
-	taste_description = "pure electricity"
+	taste_description = "energy-boosting nutrients"
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
 
-/datum/reagent/consumable/liquidelectricity/enriched
-	name = "Enriched Liquid Electricity"
+//Despite what the clown will tell you, electrolytes are not what plants crave.
+/datum/reagent/consumable/electrolyte/on_hydroponics_apply(obj/item/seeds/myseed, datum/reagents/chems, obj/machinery/hydroponics/mytray, mob/user)
+	. = ..()
+	if(chems.has_reagent(src, 1))
+		mytray.adjustHealth(-round(chems.get_reagent_amount(src.type) * 1))
+
+/datum/reagent/consumable/electrolyte/on_mob_life(mob/living/carbon/M, delta_time, times_fired)
+	if(isethereal(M))
+		M.blood_volume += 1 * delta_time
```

Comment:
or rather, make it work like actual blood does so you cant gain more blood from sucking your own blood.

Reply:
As I said in discord, since you seem to have a better idea of how this could be fixed I'll just close this pr I didn't get approval for and let you handle it since you've got authority for it now that it has your attention.